lib/cisco_node_utils/cmd_ref/vxlan_vtep.yaml

```diff
@@ -0,0 +1,26 @@
+# vxlan_vtep
+---
+all_interfaces:
+  multiple:
+  config_get: 'show running-config interface all'
+  config_get_token: '/^interface\s+(nve.*)$/'
+
+mac_distribution:
+  config_get: 'show running-config interface all'
+  config_get_token: ['/^interface <name>$/', '/^host-reachability protocol (\S+)/']
+  config_set: ['interface <name>', '<no_cmd> host-reachability protocol <proto>', 'end']
+
+#TODO: move to vxlan_vtp_vni.rb
+#member_vni_mgrp:
+#  config_get: 'show running-config interface all'
+#  config_get_token: ['/^interface <name>$/', '/^member vni (\S+)/', '/^mcast-group (\S+)$/']
+#  config_set: ['interface <name>', '<no_cmd> member vni <vni>', 'mcast-group <mcast_grp>','end']
+
+member_vni_arp:
+  config_set: ['interface <name>', '<no_cmd> member vni <vni>', 'suppress-arp']
+
+source_intf:
+  config_get: 'show running-config interface'
+  config_get_token: ['/^interface <name>$/','/^source\-interface (\S+)$/']
+  config_set: ['interface <name>', '<no_cmd> source-interface <lpbk_intf>', 'end']
```

Comment:
Because it's restricted to loopback interfaces only :). I was thrown by this as well, but it does not work with a regular 'ol' Ethernet int.